qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -55,23 +55,41 @@ def run_cephfs_shell_cmd(self, cmd, mount_x=None, opts=None, stdin=None,
 
         log.info("Running command: {}".format(" ".join(args)))
         return mount_x.client_remote.run(args=args, stdout=BytesIO(),
-                                         stderr=BytesIO(), stdin=stdin)
+                                         stderr=BytesIO(), stdin=stdin,
+                                         check_status=check_status)
+
+    def negtest_cephfs_shell_cmd(self, cmd, mount_x=None, opts=None,
+                                 stdin=None, config_path=None, retval=None,
+                                 errmsg=None):
```

Comment:
Can you rename this test to something like `check_cephfs_shell_cmd_ret_val`?

Reply:
`check_cephfs_shell_cmd_ret_val` doesn't describe it fully. The method intends to run negative test on given cephfs-shell command, so IMO, current name is fine.

Reply:
Sorry, even `negtest_cephfs_shell_cmd`  is not clear. How about `verify_cephfs_shell_failed_cmd_ret_val` ?

Reply:
Also please add log.

Reply:
> Sorry, even negtest_cephfs_shell_cmd is not clear.

Usually, when we say `test_cephfs_shell_cmd` we mean test positively (because we abort on error). I hope that explains the name "negtest". I'll add a note there since it's not clear in your opinion.

> How about verify_cephfs_shell_failed_cmd_ret_val or check_failed_cmd_ret_val ?

Again only describes it partially.